judger/src/prelude/flowsnake.rs

```diff
@@ -103,18 +103,34 @@ impl FlowSnake {
         }
         Ok(())
     }
+
+    pub fn write_str_dashed_buffered(&self, buf: &mut [u8]) -> Result<(), ()> {
```

Comment:
Now that `bool.then()` is stable, there is no point in using `Result<(), ()>` here.

Reply:
Done. See the newer commit.